### PR TITLE
Add a note how to set DALI_EXTRA_PATH to run jupyter examples

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It adds a note how to set DALI_EXTRA_PATH to run jupyter examples

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     Adds an extra note how to set DALI_EXTRA_PATH in all examples using it
 - Affected modules and functionalities:
     docs examples
 - Key points relevant for the review:
     docs examples
 - Validation and testing:
     NA
 - Documentation (including examples):
     all examples using DALI_EXTRA_PATH

**JIRA TASK**: *[DALI-1262]*

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by klecki** [comment]
I would write something like:
`DALI_EXTRA_PATH` _environment variable_ should point to the place where data from DALI extra repository is downloaded. Please make sure that the proper release tag is checked out.

And maybe swap place to directory but both indicate the dir above the test data and I don't know how to phrase it better.

**Comment by JanuszL** [comment]
> I would write something like:
> `DALI_EXTRA_PATH` _environment variable_ should point to the place where data from DALI extra repository is downloaded. Please make sure that the proper release tag is checked out.
> 
> And maybe swap place to directory but both indicate the dir above the test data and I don't know how to phrase it better.

Fixed.

**Comment by JanuszL** [comment]
!build

File: docs/examples/general/data_loading/dataloading_lmdb.ipynb [hunk 1]
```diff
@@ -14,7 +14,9 @@
     "\n",
     "In order to use data stored in LMDB in Caffe format, we need to use `CaffeReader` operator. Besides arguments common to all readers (like `random_shuffle`), it takes `path` argument - path to the directory where LMDB is stored.\n",
     "\n",
-    "Let us define a simple pipeline that takes images stored in Caffe format, decodes them and prepares them for ingestion in DL framework (crop, normalize and NHWC -> NCHW conversion)."
+    "Let us define a simple pipeline that takes images stored in Caffe format, decodes them and prepares them for ingestion in DL framework (crop, normalize and NHWC -> NCHW conversion).\n",
```
Comment (mzient) [inline docs/examples/general/data_loading/dataloading_lmdb.ipynb]:
```suggestion
    "Let us define a simple pipeline that takes images stored in Caffe format, decodes them and prepares them for ingestion in DL framework (crop, normalize and HWC -> CHW conversion).\n",
```

File: docs/examples/sequence_processing/video/video_reader_label_example.ipynb [hunk 2]
```diff
@@ -49,7 +49,7 @@
...
    "source": [
-    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is a mp4 video container containing a h264 and under the Create Common license. We've split it into 5 second clips and divided the clips into labelled groups. This can be done easily with the `ffmpeg` standalone tool. Our prepared data can be found in [DALI_extra](https://github.com/NVIDIA/DALI_extra) repository. Snippet below verifies, that you have defined `DALI_extra` path as environment variable."
+    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is a mp4 video container containing a h264 and under the Create Common license. We've split it into 5 second clips and divided the clips into labelled groups. This can be done easily with the `ffmpeg` standalone tool. `DALI_EXTRA_PATH` environment variable should point to the place where data from [DALI extra repository](https://github.com/NVIDIA/DALI_extra) is downloaded. Please make sure that the proper release tag is checked out. Snippet below verifies, that you have defined `DALI_extra` path as environment variable."
```
Comment (mzient) [inline docs/examples/sequence_processing/video/video_reader_label_example.ipynb]:
```suggestion
    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is an mp4 container containing an h.264 video and distributed under the Create Common license. We've split it into 5 second clips and divided the clips into labelled groups. This can be done easily with the `ffmpeg` standalone tool. `DALI_EXTRA_PATH` environment variable should point to the place where data from [DALI extra repository](https://github.com/NVIDIA/DALI_extra) is downloaded. Please make sure that the proper release tag is checked out. The snippet below verifies, that you have defined `DALI_extra` path as an environment variable."
```

File: docs/examples/sequence_processing/video/video_reader_simple_example.ipynb [hunk 3]
```diff
@@ -49,7 +49,7 @@
...
+    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is a mp4 video container containing a h264 and under the Create Common license. Let's split it into 10s clips in order to check how `VideoReader` handles mutliple video files. This can be done easily with the `ffmpeg` standalone tool. `DALI_EXTRA_PATH` environment variable should point to the place where data from [DALI extra repository](https://github.com/NVIDIA/DALI_extra) is downloaded. Please make sure that the proper release tag is checked out. Snippet below verifies, that you have defined `DALI_extra` path as environment variable."
```
Comment (mzient) [inline docs/examples/sequence_processing/video/video_reader_simple_example.ipynb]:
```suggestion
    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is an mp4 container containing an h.264 video and distributed under the Create Common license. We've split it into 5 second clips and divided the clips into labelled groups. This can be done easily with the `ffmpeg` standalone tool. `DALI_EXTRA_PATH` environment variable should point to the place where data from [DALI extra repository](https://github.com/NVIDIA/DALI_extra) is downloaded. Please make sure that the proper release tag is checked out. The snippet below verifies, that you have defined `DALI_extra` path as an environment variable."
```
```

File: docs/examples/frameworks/tensorflow/tensorflow-plugin-tutorial.ipynb [hunk 4]
```diff
@@ -25,7 +25,9 @@
     "\n",
     "First we start by defining some parameters for DALI and Tensorflow.\n",
     "\n",
-    "We will use a subsample of Imagenet stored in a MXNet's RecordIO for this tutorial. For details on how to use `MXNetReader` as well as other readers please see other [examples](../../index.rst)."
+    "We will use a subsample of Imagenet stored in a MXNet's RecordIO for this tutorial. For details on how to use `MXNetReader` as well as other readers please see other [examples](../../index.rst).\n",
```
Comment (mzient) [inline docs/examples/frameworks/tensorflow/tensorflow-plugin-tutorial.ipynb]:
```suggestion
    "In this tutorial, we will use a subsample of Imagenet stored in an MXNet's RecordIO. For details on how to use `MXNetReader`, as well as other readers, please see other [examples](../../index.rst).\n",
```

File: docs/examples/frameworks/tensorflow/tensorflow-plugin-sparse-tensor.ipynb [hunk 5]
```diff
@@ -21,7 +21,9 @@
     "### Defining the data loading pipeline\n",
     "\n",
     "First, we start by defining some simple pipeline that will return data as a sparse tensor. To ochieve this, we will use well known COCO data set. Each image may have 0 or more bounding boxes with labels describing objects present in it.Wa want to return images in a normalized way, while labels and bounding boxes will be represented as sparse tensors.\n",
-    "At the begginig let us define some global parameters"
+    "At the begginig let us define some global parameters\n",
```
Comment (mzient) [inline docs/examples/frameworks/tensorflow/tensorflow-plugin-sparse-tensor.ipynb]:
```suggestion
    "At the beginning, let us define some global parameters\n",
```

File: docs/examples/frameworks/tensorflow/tensorflow-dataset-multigpu.ipynb [hunk 6]
```diff
@@ -10,7 +10,9 @@
     "\n",
     "This notebook is a comprehensive example on how to use DALI `tf.data.Dataset` with multiple GPUs. It is recommended to look into [single GPU example](tensorflow-dataset.ipynb) first to get up to speed with DALI dataset and how it can be used to train a neural network with custom model and training loop. This example is an extension of the single GPU version.\n",
     "\n",
-    "Initially we define some parameters of the training and to create a DALI pipeline to read [MNIST](http://yann.lecun.com/exdb/mnist/) converted to LMDB format. You can find it in [DALI_extra](https://github.com/NVIDIA/DALI_extra) dataset. This pipeline is able to shard dataset into multiple parts."
+    "Initially we define some parameters of the training and to create a DALI pipeline to read [MNIST](http://yann.lecun.com/exdb/mnist/) converted to LMDB format. You can find it in [DALI_extra](https://github.com/NVIDIA/DALI_extra) dataset. This pipeline is able to shard dataset into multiple parts.\n",
```
Comment (mzient) [inline docs/examples/frameworks/tensorflow/tensorflow-dataset-multigpu.ipynb]:
```suggestion
    "Initially we define some parameters of the training and to create a DALI pipeline to read [MNIST](http://yann.lecun.com/exdb/mnist/) converted to LMDB format. You can find it in [DALI_extra](https://github.com/NVIDIA/DALI_extra) dataset. This pipeline is able to partition the dataset into multiple shards.\n",
```

File: docs/examples/frameworks/tensorflow/idx_files/train.idx [hunk 7]
```diff
@@ -0,0 +1,47 @@
+0 171504
```
Comment (jantonguirao) [inline docs/examples/frameworks/tensorflow/idx_files/train.idx]: is this file meant to be part of the PR?

Comment (JanuszL) [reply docs/examples/frameworks/tensorflow/idx_files/train.idx]: Leftover. Fixed

File: docs/examples/frameworks/tensorflow/tensorflow-dataset-multigpu.ipynb (hunk 6 above)
Comment (JanuszL) [reply docs/examples/frameworks/tensorflow/tensorflow-dataset-multigpu.ipynb]: Legacy. Fixed

File: docs/examples/frameworks/tensorflow/tensorflow-plugin-sparse-tensor.ipynb (hunk 5 above)
Comment (JanuszL) [reply docs/examples/frameworks/tensorflow/tensorflow-plugin-sparse-tensor.ipynb]: Legacy. Fixed

File: docs/examples/frameworks/tensorflow/tensorflow-plugin-tutorial.ipynb (hunk 4 above)
Comment (JanuszL) [reply docs/examples/frameworks/tensorflow/tensorflow-plugin-tutorial.ipynb]: Legacy. Fixed

File: docs/examples/general/data_loading/dataloading_lmdb.ipynb (hunk 1 above)
Comment (JanuszL) [reply docs/examples/general/data_loading/dataloading_lmdb.ipynb]: Legacy. Fixed

File: docs/examples/sequence_processing/video/video_reader_label_example.ipynb (hunk 2 above)
Comment (JanuszL) [reply docs/examples/sequence_processing/video/video_reader_label_example.ipynb]: Legacy. Fixed

File: docs/examples/sequence_processing/video/video_reader_simple_example.ipynb (hunk 3 above)
Comment (JanuszL) [reply docs/examples/sequence_processing/video/video_reader_simple_example.ipynb]: Legacy. Fixed
